client/signup/jetpack-connect/plans-grid.jsx

```diff
@@ -10,22 +11,38 @@ import Main from 'components/main';
 import StepHeader from '../step-header';
 import PlansFeaturesMain from 'my-sites/plans-features-main';
 
-export default React.createClass( {
-	displayName: 'JetpackPlansGrid',
+class JetpackPlansGrid extends Component {
+	static propTypes = {
+		basePlansPath: PropTypes.string.isRequired,
```

Comment:
Why required? `PlansFeaturesMain` appears to not require `basePlansPath` to be required.